lib/active_model/serializer/adapter/base.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      Base = ActiveModelSerializers::Adapter::Base
```

Comment:
I wasn't sure how to properly handle the base class to properly allow inheritance and add warning at the same time.